.github/workflows/documentation.yml

```diff
@@ -0,0 +1,41 @@
+name: documentation
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  gh-release:
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Add key to allow access to repository
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+        run: |
+          mkdir -p ~/.ssh
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          echo "${{ secrets.GH_PAGES_DEPLOY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          cat <<EOT >> ~/.ssh/config
+          Host github.com
+          HostName github.com
+          IdentityFile ~/.ssh/id_rsa
+          EOT
+      - name: Release to GitHub Pages
+        env:
+          USE_SSH: true
+          GIT_USER: git
+        run: |
+          git config --global user.email "application-services@mozilla.com"
+          git config --global user.name "Application Services"
+          yarn install --frozen-lockfile
+          npm ci
```

Comment:
I think we're [missing a few conditionals](https://v2.docusaurus.io/docs/deployment#triggering-deployment-with-github-actions) in this script and it's [causing an error](https://github.com/mozilla/experimenter-docs/runs/2008160447?check_suite_focus=true). (Or at least, the "check" job failed on this, but I don't see that in this action any more)

Looks like it needs the checks for yarn.lock vs package-lock.json to run just yarn or npm as needed:
```
          if [ -e yarn.lock ]; then
          yarn install --frozen-lockfile
          elif [ -e package-lock.json ]; then
          npm ci
          else
          npm i
          fi
          npx docusaurus deploy
```

Reply:
:hehehe: Whoops, I thought I had remove the NPM stuff since we are using Yarn, guess I missed some. Thanks for catching!

Reply:
Oh also, there used to be a checks workflow that I removed, that's what that failing workflow you linked to is from

Reply:
Yeah, we might actually want that checks workflow just to sanity check PRs since there is a build process here